app/components/validated-input-group.hbs

```diff
@@ -0,0 +1,17 @@
+<label>
+    <div class="text">
+        {{@label}}
+    </div>
+
+    <Input
+        @focus-out={{this.onFocusOut}}
```

Comment:
I suggest placing important params first, value and type are more important then focus out and can be moved to the top for example